src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -600,6 +600,17 @@ public override bool Equals(object obj)
         }
```

Comment:
Can we mark this implementation `sealed` if implemented as:
```
return this.Equals(obj as Symbol,
    SymbolEqualityComparer.Default.CompareKind);
```

Reply:
Discussed this offline. Agree we should do this, but its quite a large (albeit mechanical) change. Have filed [https://github.com/dotnet/roslyn/issues/37409](https://github.com/dotnet/roslyn/issues/37409) as a follow up issue 

---
In reply to: [305552205](https://github.com/dotnet/roslyn/pull/37247#discussion_r305552205) [](ancestors = 305552205)